cypress/integration/taskCloneTableColumn.spec.js

```diff
@@ -1,33 +1,24 @@
 'use strict';
 
 describe('Clone table app', () => {
-  before(() => {
+  beforeEach(() => {
     cy.visit('/');
+    cy.get('th:nth-child(2)').debug();
   });
 
   it('should only change number of columns in original table', () => {
     cy.get('table').children().should('have.length', 3);
-    cy.get('thead > tr').children().should('have.length', 6);
+    cy.get('thead > tr > th').should('have.length', 6);
```

Comment:
The assertion for the number of `th` elements might fail if the table layout changes. It's better to check for a minimum number of headers expected rather than an exact number unless the exact count is a requirement.